tests/unit/Statement/StatementListTest.php

```diff
@@ -17,9 +17,66 @@
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @author Thiemo Mättig
  */
 class StatementListTest extends \PHPUnit_Framework_TestCase {
 
+	/**
+	 * @param int $propertyId
+	 * @param string|null $guid
+	 * @param int $rank
+	 *
+	 * @return Statement
+	 */
+	private function getStatement( $propertyId, $guid, $rank = Statement::RANK_NORMAL ) {
```

Comment:
Personally I really dislike the `get...Mock` naming scheme. Code that calls this should not care if it gets a mock or a real object. As the `@return` tag states.